stubs/python-xlib/Xlib/X.pyi

```diff
@@ -1,345 +1,347 @@
-FillTiled: int
-FillStippled: int
-FillOpaqueStippled: int
-EvenOddRule: int
-WindingRule: int
-ClipByChildren: int
-IncludeInferiors: int
-Unsorted: int
-YSorted: int
-YXSorted: int
-YXBanded: int
-CoordModeOrigin: int
-CoordModePrevious: int
-Complex: int
-Nonconvex: int
-Convex: int
-ArcChord: int
-ArcPieSlice: int
-GCFunction: int
-GCPlaneMask: int
-GCForeground: int
-GCBackground: int
-GCLineWidth: int
-GCLineStyle: int
-GCCapStyle: int
-GCJoinStyle: int
-GCFillStyle: int
-GCFillRule: int
-GCTile: int
-GCStipple: int
-GCTileStipXOrigin: int
-GCTileStipYOrigin: int
-GCFont: int
-GCSubwindowMode: int
-GCGraphicsExposures: int
-GCClipXOrigin: int
-GCClipYOrigin: int
-GCClipMask: int
-GCDashOffset: int
-GCDashList: int
-GCArcMode: int
-GCLastBit: int
-FontLeftToRight: int
-FontRightToLeft: int
-FontChange: int
-XYBitmap: int
-XYPixmap: int
-ZPixmap: int
-AllocNone: int
-AllocAll: int
-DoRed: int
-DoGreen: int
-DoBlue: int
-CursorShape: int
-TileShape: int
-StippleShape: int
-AutoRepeatModeOff: int
-AutoRepeatModeOn: int
-AutoRepeatModeDefault: int
-LedModeOff: int
-LedModeOn: int
-KBKeyClickPercent: int
-KBBellPercent: int
-KBBellPitch: int
-KBBellDuration: int
-KBLed: int
-KBLedMode: int
-KBKey: int
-KBAutoRepeatMode: int
-MappingSuccess: int
-MappingBusy: int
-MappingFailed: int
-MappingModifier: int
-MappingKeyboard: int
-MappingPointer: int
-DontPreferBlanking: int
-PreferBlanking: int
-DefaultBlanking: int
-DisableScreenSaver: int
-DisableScreenInterval: int
-DontAllowExposures: int
-AllowExposures: int
-DefaultExposures: int
-ScreenSaverReset: int
-ScreenSaverActive: int
-HostInsert: int
-HostDelete: int
-EnableAccess: int
-DisableAccess: int
-StaticGray: int
-GrayScale: int
-StaticColor: int
-PseudoColor: int
-TrueColor: int
-DirectColor: int
-LSBFirst: int
-MSBFirst: int
+from typing_extensions import Final
+
+NONE: Final = 0
+ParentRelative: Final = 1
+CopyFromParent: Final = 0
+PointerWindow: Final = 0
+InputFocus: Final = 1
+PointerRoot: Final = 1
+AnyPropertyType: Final = 0
+AnyKey: Final = 0
+AnyButton: Final = 0
+AllTemporary: Final = 0
+CurrentTime: Final = 0
+NoSymbol: Final = 0
+NoEventMask: Final = 0
+KeyPressMask: Final = 1
+KeyReleaseMask: Final = 2
+ButtonPressMask: Final = 4
+ButtonReleaseMask: Final = 8
+EnterWindowMask: Final = 16
+LeaveWindowMask: Final = 32
+PointerMotionMask: Final = 64
+PointerMotionHintMask: Final = 128
+Button1MotionMask: Final = 256
+Button2MotionMask: Final = 512
+Button3MotionMask: Final = 1024
+Button4MotionMask: Final = 2048
+Button5MotionMask: Final = 4096
+ButtonMotionMask: Final = 8192
+KeymapStateMask: Final = 16384
+ExposureMask: Final = 32768
+VisibilityChangeMask: Final = 65536
+StructureNotifyMask: Final = 131072
+ResizeRedirectMask: Final = 262144
+SubstructureNotifyMask: Final = 524288
+SubstructureRedirectMask: Final = 1048576
+FocusChangeMask: Final = 2097152
+PropertyChangeMask: Final = 4194304
+ColormapChangeMask: Final = 8388608
+OwnerGrabButtonMask: Final = 16777216
```

Comment:
If we used octal or hexadecimal for these, it might be more readable -- it would be more obvious that these are all powers of 2.

(Similar for a bunch of other constants in this file.)

Reply:
Ping on this, I do agree that hex would probably work better here.